### ISSUE TITLE
Should it be possible for the user to allow the func_time to be greater than the total window time?

### ISSUE THREAD

**Issue by Bachibouzouk** [RAMP-project/RAMP#37]
I came across a situation where the code was stopping due to an error (the `max(free_spots)` was returning a `ValueError` because `free_spots` was an empty array).

Looking at it closely I noticed that this happened for an appliance for which the cumulative time defined within windows was smaller than the provided `func_time`. I wrote a small check to verify that upon calling of the `windows` method and came to see that in the `input_file_1.py` there was such a problem with the appliance `Ch_speaker`
```
Ch_speaker = Church.Appliance(Church,1,100,1,240,0.2,60)
Ch_speaker.windows([1200,1350],[0,0],0.1)
```
Here the `func_time` is 240min and the window allows for 150 min only

I just wanted to know what should be expected/wished in that case?

**Comment by Bachibouzouk** [comment]
@FLomb , @mohammadamint

**Comment by FLomb** [comment]
I have noticed the issue too, and have fixed it. I am inclined to believe that this was a typo and that the intended window length was 150 min.

**Comment by Bachibouzouk** [comment]
I would not considered the issue as closed as the fix was done to the input file, it is therefore still possible to wrongfully input func_time greater than window time without being notified about it

**Comment by FLomb** [comment]
Actually, the development branch with your new pushes includes a check [line 529 of the core.py file](https://github.com/RAMP-project/RAMP/blob/development/ramp/core/core.py#L529) that prevents this from happening. This is indeed how I noticed the typo in the example input file 1, which I have corrected accordingly. So all seems consistent in the development branch. Sure, the issue is still there in the master branch until it does not get updated.
